frontend/packages/ceph-storage-plugin/src/components/independent-mode/utils.ts

```diff
@@ -27,3 +29,30 @@ export const checkForIndependentSupport = (csv: ClusterServiceVersionKind): bool
     csv.metadata.annotations?.['external.cluster.ocs.openshift.io/supported'];
   return independent === 'true';
 };
+
+enum ClusterPhase {
+  CONNECTED = 'Connected',
+  READY = 'Ready',
+  CONNECTING = 'Connecting',
+  PROGRESSING = 'Progressing',
+  ERROR = 'Error',
+}
+
+const PhaseToState = Object.freeze({
+  [ClusterPhase.CONNECTED]: HealthState.OK,
+  [ClusterPhase.READY]: HealthState.OK,
+  [ClusterPhase.CONNECTING]: HealthState.UPDATING,
+  [ClusterPhase.PROGRESSING]: HealthState.UPDATING,
+  [ClusterPhase.ERROR]: HealthState.ERROR,
+});
+
+export const getClusterHealth = (cluster: K8sResourceKind, loaded: boolean, error): HealthState => {
+  const phase = cluster?.status?.phase;
+  if (!_.isEmpty(error)) {
+    if (error?.response?.status === 404) return HealthState.UNKNOWN;
```

Comment:
```suggestion
    if (error?.response?.status === 404) return HealthState.NOTAVAILABLE;
```

Reply:
But OCS Operator is available the storage cluster endpoint is not available,

Reply:
I think it will be the status of independent mode cluster. We need not to see wrt operator status, whose status is shown on overview status card